code/game/objects/structures/flora.dm

```diff
@@ -78,7 +78,31 @@
 	desc = "Oh, no. Not again"
 	icon = 'icons/obj/plants.dmi'
 	icon_state = "plant-26"
-
+	layer = FLY_LAYER
+	var/full = null

```

Comment:
This should be a typed var, like var/atom/movable/full, or var/obj/item/full. This lets you access its vars, like loc.